lib/import/helpers/brca/providers/rj1/rj1_ashkenazi_test_processor.rb

```diff
@@ -0,0 +1,60 @@
+module Import
+  module Helpers
+    module Brca
+      module Providers
+        module Rj1
+          # Processor for Ashkenazi Screen Tests exctraction methods
+          module Rj1AshkenaziTestProcessor
+            include Import::Helpers::Brca::Providers::Rj1::Rj1Constants
+
+            #####################################################################################
+            ################ HERE ARE ASHKENAZI TESTS ###########################################
+            #####################################################################################
+
+            def add_ajscreen_date
+              return if @aj_report_date.nil? && @report_report_date.nil?
+
+              if @aj_report_date.present?
+                @genotype.attribute_map['authoriseddate'] = @aj_report_date
+              elsif @report_report_date.present?
+                @genotype.attribute_map['authoriseddate'] = @predictive_report_date
+              end
+            end
+
+            def normal_ashkenazi_test?
+              return if @aj_assay_result.nil?
+
+              @aj_assay_result.scan(/neg|nrg/i).size.positive? &&
+                @brca1_mutation.nil? && @brca2_mutation.nil?
+            end
+
+            def positive_ashkenazi_test?
+              return if @aj_assay_result.nil?
```

Comment:
Can [predicate](http://ruby-for-beginners.rubymonstas.org/objects/predicates.html) methods please return `true` or `false`. This guard clause will return `nil`. There are a few examples.